NOTE: /usr/share/pixmaps/fedora-logo-small.png is is save to use as it is
provided both by fedora-logos and generic-logos. If both are missing, the
greeter will fall back to the default 'computer' icon from data/greeter.ui.
Therefor, we don't need to add any additional dependencies.

--- lightdm-gtk-greeter-1.3.1.orig/data/lightdm-gtk-greeter.conf	2012-08-23 19:29:44.000000000 +0200
+++ lightdm-gtk-greeter-1.3.1/data/lightdm-gtk-greeter.conf	2012-10-03 11:32:46.769147108 +0200
@@ -11,8 +11,8 @@
 # show-language-selector (true or false)
 #
 [greeter]
-#logo=
-#background=
+logo=/usr/share/pixmaps/fedora-logo-small.png
+background=/usr/share/backgrounds/default.png
 #theme-name=
 #icon-theme-name=
 #font-name=
